package.json

```diff
@@ -12,11 +12,12 @@
     "prepublishOnly": "npm run swagger",
     "pretest": "npm run swagger",
     "test": "npm run cover && npm run report",
+    "test:debug": "mocha --no-timeouts test/",
     "posttest": "npm run lint",
     "report": "nyc report --reporter=lcov",
     "lint": "eslint-godaddy -c .eslintrc \"{lib,test}/**/*.js\"",
     "diagrams": "wrhs-diagrams --source=assets/diagrams --target=assets",
-    "localstack": "docker run -it -p 4569:4569 -p 4572:4572 --rm localstack/localstack",
+    "localstack": "docker run -it -p 4566:4566 --rm localstack/localstack",
```

Comment:
should this be a pinned version? (also if you pin this version you don't need to do the `docker pull` [here](https://github.com/godaddy/warehouse.ai/pull/94/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R475)

Reply:
Yeah it should.